sct.py

```diff
@@ -1755,6 +1760,24 @@ def finish_argus_test_run(jenkins_status):
         LOGGER.error("Failed to submit data to Argus", exc_info=True)
 
 
+@cli.command("fetch-junit-form-runner", help="copy the junit.xml back")
+@click.argument("runner-ip", type=str)
+def fetch_junit(runner_ip):
```

Comment:
Why so specific func?
Why not "fetch_file"? So it could be reused elsewhere...

Reply:
one can refactor it, when there would be a need, there isn't a need